server/storage/players.lua

```diff
@@ -219,7 +221,8 @@ local function fetchPlayerEntity(citizenId)
         job = player.job and json.decode(player.job),
         gang = player.gang and json.decode(player.gang),
         position = convertPosition(player.position),
-        metadata = json.decode(player.metadata)
+        metadata = json.decode(player.metadata),
+        lastLoggedOut = player.lastLoggedOut/1000
```

Comment:
ditto

Reply:
This is different than fetching all player entities? Also you need to math.floor the division to guarantee that it won't be a decimal number

Reply:
My bad for the difference,
Still no need to floor it tho but ok.